### PR TITLE
NO AUTO Moves nodes, create, refactor, trigger tests to core only

### PR DESCRIPTION
**Pull request by ncordon**
Strips core `nodes`, `create`, `refactor` and `trigger` procedures and functions from full package.

To be merged after #3087 .

### CODE REVIEW

File: core/src/test/resources/csv-inputs/forum.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+id:ID(Forum)|title:STRING|creationDate:LONG
+0|Wall of Mahinda Perera|20100214153220447
```
Comment (AzuObs) [inline core/src/test/resources/csv-inputs/forum.csv:2]: Is the file "forum.csv" used? I can do a search and find "personsWithoutIdField.csv" but not this one

**Review by AzuObs — APPROVED** [review]
Nice work 💯 

One small comment you may want to check, but feel free to merge even if you think a small change was needed

File: core/src/test/resources/csv-inputs/forum.csv (hunk 1 above)
Comment (ncordon) [reply core/src/test/resources/csv-inputs/forum.csv]: This is leftover from having built this PR on top of another. It shouldn't be there anymore. This file is generated automatically by some tests.

File: build.gradle [hunk 2]
```diff
@@ -79,7 +79,6 @@ subprojects {
         forkEvery = 50
         maxParallelForks = 1
         minHeapSize = "128m"
-        maxHeapSize = "512m"
```
Comment (ncordon) [inline build.gradle:82]: Funny cause this option is set to 8G just 4 lines above, so I wondered which one prevailed.

**Pull request merged**